solutions/bronze/cf-1624D.mdx

```diff
@@ -0,0 +1,120 @@
+---
+id: cf-1624D
+source: CF
+title: Palindromes Coloring
+author: Rameez Parwez
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/98942)
+
+## Explanation
+
+We can greedily solve this problem by first forming palindromic pairs from identical characters.
+We add one pair each of the `k` palindromes we want to build, as long as we have `k` pairs available.
+Once we are done adding pairs, we can still add one more character to the middle of each palindrome if there are atleast `k` individual characters remaining.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using std::cout;
```

Comment:
just curious but why only have `using std::cout`? idk doesnt matter that much but seems a bit odd to me

Reply:
sans way of writing.

changed it.